lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -35,7 +32,7 @@
                 "at the {link_to_provider_site} website. Select <b>Request Credit</b> to get started.").format(
                     link_to_provider_site=provider_link,
                 )
-                credit_msg_class = "credit-request-pending-msg"
+                credit_msg_class = "credit-request-not-started-msg"
```

Comment:
`credit_msg_class` is not actually used for styling. It's a sad hack for unit testing...that I don't feel like cleaning up right now. `credit_btn_class` is not-so-great way of triggering JS that will create a credit request when the value is set to `pending-credit-btn`. Now do you understand what I don't like working with this code?